recipes/ann/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ann" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "eea03f2e224b66813226d775053316675375dcec45bd263674c052d9324a49a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: http://www.cs.umd.edu/~mount/ANN/Files/{{ version }}/{{ name }}_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/cmake.patch
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py36]
+  track_features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py36]
```

Comment:
Do you really need those ``features`` and ``track_features`` ?

Reply:
Not really, I have removed them